tests/basic.t

```diff
@@ -65,49 +66,125 @@ if [ $? -eq 1 ]; then
   TEST glusterd
 fi
 
-# Create gluster volume
+##### Create Gluster Volume Test Start #####
 TEST gluster vol create ${VOLNAME} ${HOST}:${BRKDIR} force
 
 # Start the volume
 TEST gluster vol start ${VOLNAME}
+##### End #####
 
 
-# Test gfapi access
+##### Gfapi Access Test Start #####
 TEST ./tests/gfapi-test ${VOLNAME} ${HOST}
+##### End #####
 
-# Start gluster-blockd.service
+
+##### Restart gluster-blockd.service Test Start #####
 systemctl daemon-reload
 TEST systemctl restart gluster-blockd.service
-sleep 1;
+sleep 5
+##### End #####
+
+
+##### gluster-block 'version' and 'help' Test Start #####
+TEST gluster-block version
+TEST gluster-block help
+##### End #####
+
+
+##### Block 'create' and 'delete' Test Start #####
+# Simple block create/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Simple block create/delete with 'timeout' set
+TEST gluster-block timeout 350 create ${VOLNAME}/${BLKNAME} ${HOST} 1MiB
+TEST gluster-block timeout 350 delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'auth enable' set/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} auth enable ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'auth disable' set/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} auth disable ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'ha' set/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} ha 1 ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'prealloc full' set/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} prealloc full ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'prealloc no' set/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} prealloc no ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'ring-buffer' set/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} ring-buffer 32 ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'storage' set and delete with 'unlink-storage no' set
+TEST gluster-block create ${VOLNAME}/${BLKNAME} ${HOST} 1MiB
+LINK=`eval gluster-block info ${VOLNAME}/${BLKNAME} | grep GBID | awk -F: '{print $2}'`
```

Comment:
The delimiter should be -F' ' (space)

Reply:
May be awk -F' ' '{print $NF}' is better

Reply:
make sense.